src/lib/connectors/repos/queries/gauging-stations.js

```diff
@@ -0,0 +1,15 @@
+exports.findGaugingStations = `select 
```

Comment:
Could you rename this query to something more detailed, such as `findGaugingStationWithLinkedLicences` or something like that - just because it's not actually finding gauging stations which is what the current name suggests